modules/ibi-create-standalone-config-iso.adoc

```diff
@@ -72,6 +73,14 @@ pullSecret: '{"auths":{"<your_pull_secret>"}}}'
 sshKey: 'ssh-rsa <your_ssh_pub_key>'
 ----
 
+[IMPORTANT]
+====
+If your cluster deployment requires a proxy configuration, you must do the following:
+* Create a seed image from a seed cluster featuring a proxy configuration. The proxy configurations do not have to match.
+* Configure the machineNetwork field in your installation manifest.
```

Comment:
```suggestion
If your cluster deployment requires a proxy configuration, you must do the following:

* Create a seed image from a seed cluster featuring a proxy configuration. The proxy configurations do not have to match.
* Configure the machineNetwork field in your installation manifest.
```

Reply:
![image](https://github.com/user-attachments/assets/af4eb89a-7ff4-4b7c-8de2-b1c85f414e12)
You need a space before the unordered list so it renders.